README.md

```diff
@@ -1,209 +1,257 @@
-# Install, Build, and Test the Extension
+# Zowe Explorer
```

Comment:
I believe that the README in `packages/zowe-explorer` is what will appear in https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe since that's what's packaged on `yarn package`.
(Please @VitGottwald or @phaumer correct me if I'm mistaken)

I do agree that we need to do something to the top-level readme to reflect the new nature of the repo and properly instruct people (mainly developers?) on what the structure looks like and where they can go to make updates, run tests, and ultimately deploy to the corresponding places.

I do like the contents though 😋

Reply:
Wait, so what do we do now? Move back the core readme to the `packages/zowe-explorer` destination?

Reply:
Yeah, I believe we will need to move this one back to the `packages/zowe-explorer` and create new content for the top level that better describes the state of the repo.
As a test, I don't mind publishing a"beta" version with these changes to the Open VSX Registry or the Marketplace to verify that the Readme is indeed required at the packages level.